operations/security_and_authentication/Restrict_Access_to_NCN_Images_S3_Bucket.md

```diff
@@ -584,7 +584,7 @@ The configuration applied in this procedure was tested against barebones images
          do
             curl_rept "curl $CURL_O ncn-s00${n}.${t}:8080" # ceph rados
             curl_rept "curl $CURL_O http://ncn-s00${n}.${t}/ncn-images/" # ncn images, http
-            curl_rept "curl $CURL_O-k  https://ncn-s00${n}.${t}/ncn-images/" # ncn images, https
+            curl_rept "curl $CURL_O -k  https://ncn-s00${n}.${t}/ncn-images/" # ncn images, https
```

Comment:
Should maybe use `-f` in each of the `curl` calls to ensure they issue a non-zero return code if they fail.

Reply:
Thanks, $CURL_O includes -f.